cert_schema/3.0-beta/context.json

```diff
@@ -0,0 +1,34 @@
+{
+  "@context": {
+    "id": "@id",
+    "type": "@type",
+    "bc": "https://w3id.org/blockcerts#",
+    "cp": "https://w3id.org/chainpoint#",
+    "schema": "http://schema.org/",
+    "sec": "https://w3id.org/security#",
+    "xsd": "http://www.w3.org/2001/XMLSchema#",
+
+    "MerkleProof2019": "sec:MerkleProof2019",
+
+    "BlockcertsCredential": "bc:BlockcertsCredential",
+    "introductionUrl": { "@id": "bc:introductionUrl", "@type": "@id" },
+
+    "CryptographicKey": "sec:Key",
+
+    "domain": "sec:domain",
+    "nonce": "sec:nonce",
+    "proofValue": "sec:proofValue",
+    "assertionMethod": {"@id": "sec:assertionMethod", "@type": "@id", "@container": "@set"},
+    "authentication": {"@id": "sec:authenticationMethod", "@type": "@id", "@container": "@set"},
+    "proofPurpose": {"@id": "sec:proofPurpose", "@type": "@vocab"},
+    "verificationMethod": {"@id": "sec:verificationMethod", "@type": "@id"},
+    "created": {"@id": "http://purl.org/dc/terms/created", "@type": "xsd:dateTime"},
+    "challenge": "sec:challenge",
+    "expires": {"@id": "sec:expiration", "@type": "xsd:dateTime"},
+
+
+    "name": {
+      "@id": "schema:name"
+    }
+  }
+}
```

Comment:
we will want to add `display` in this file

Reply:
yeah, I haven't got there yet.